js/hoot/control/utilities/ImportDirectory.js

```diff
@@ -168,10 +179,52 @@ Hoot.control.utilities.importdirectory = function(context) {
     * @desc gets file list from options in import directory file list
     **/
     var _getFilesList = function(){
-        var filesList = _.map(d3.select('#importDirectoryFilesList').selectAll('option')[0],function(opt){return opt.value;});
+        var filesList = _.map(d3.select('#importDirectoryFilesList').selectAll('option')[0],function(opt){return {value:opt.value,text:opt.text};});
         return filesList;
     };
 
+    var _checkForDescription = function(){
+        var mgcpCheck = d3.select('#importDirectorySchema').value().indexOf('MGCP');
+        var tdsCheck = d3.select('#importDirectorySchema').value().indexOf('TDS');
+        var cboxBool = mgcpCheck > -1 || tdsCheck > -1;
+        d3.select('.cboxAppendFCode').classed('hidden',!cboxBool).select('input').property('checked',false);
+        _getDescriptionList('MGCP');
+    };
+
+    var _getTranslation = function(){
+        var selectedTrans = d3.select('#importDirectorySchema').datum().combobox.filter(function(d){return d.DESCRIPTION === d3.select('#importDirectorySchema').value();});
+        var retval;
+        try{
+            retval = selectedTrans[0].NAME;
+        } catch (err) {
+            retval = 'MGCP';
+        }
+
+        return retval;
+    };
+
+    var _getDescriptionList = function(translation){
+        if(!translation){translation = _getTranslation();}
+
+        d3.xhr(window.location.protocol + '//' + window.location.hostname +
+            Hoot.model.REST.formatNodeJsPortOrPath(iD.data.hootConfig.translationServerPort)
+        +'/schema?translation='+translation+'&limit=10000')
+        .get(function(error, resp){
```

Comment:
We don't really need to make a call to the translation server when we're removing the fcode descriptions.  This class should also probably cache the response to only have to make the call once.